sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/spanner/changestreams/util/TimestampGenerator.java

```diff
@@ -34,7 +33,7 @@ public TimestampGenerator() {
 
   @Override
   public Timestamp generate(SourceOfRandomness random, GenerationStatus status) {
-    final long seconds = random.nextLong(MIN_SECONDS, MAX_SECONDS);
+    final long seconds = random.nextLong(0L, MAX_SECONDS);
```

Comment:
Why did we change this? I think that the generator should always generate from all the valid values of the domain. If you'd like to restrict it on specific tests, we can use assumptions (https://pholser.github.io/junit-quickcheck/site/1.0/usage/constraining.html). I think we will probably have to extend this generator to accept configuration as well.

Reply:
Nevertheless, IMHO we should keep testing with MIN to MAX seconds

Reply:
Will revert this change and assume timestamps starting from `Epoch` in the actual tests.